### PR TITLE
deleted all the create-react-app stuff i dont think i need, added MUI…

### PR DESCRIPTION
**Pull request by gmd298**
…, and started building Nav component... which is still broken

### CODE REVIEW

**Comment by gmd298** [comment]
Did  this work ?

File: src/Nav.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
+import * as React from 'react';
```
Comment (JakeBent) [inline src/Nav.js:1]: this is boilerplat from MUI?

File: src/index.js [hunk 2]
```diff
@@ -1,8 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
```
Comment (JakeBent) [inline src/index.js:3]: why did you remove this?

**Review by JakeBent — CHANGES_REQUESTED** [review]
You should make a commit that doesn't delete anything. Just add MUI

File: src/Nav.js (hunk 1 above)
Comment (gmd298) [reply src/Nav.js:1]: Yeah it is. Thats one of the things I wanted to ask you about. I've never seen it before. Looks pretty important though.

File: src/index.js (hunk 2 above)
Comment (gmd298) [reply src/index.js:3]: I removed it because MUI is basically like inline css and I didnt think i would need a index.css file.
